daemon/execdriver/windows/terminatekill.go

```diff
@@ -26,7 +26,7 @@ func kill(id string, pid int) error {
 
 	// Terminate Process
 	if err = hcsshim.TerminateProcessInComputeSystem(id, uint32(pid)); err != nil {
-		logrus.Warnf("Failed to terminate pid %d in %s", id, pid, err)
+		logrus.Warnf("Failed to terminate pid %d in %s %q", pid, id, err)
```

Comment:
you should probably add a ":" or "-" between %s and %q just so its easier to read.